docs/docs/getting_started/cli.md

```diff
@@ -21,18 +21,18 @@ If a subdirectory for a given namespace does not exist, Synth will create it.
 #### Argument
 
 - `<namespace>` - The path to the namespace directory into which to save schema files. The directory will be created by `synth`.
-  
+
 #### Options
- 
-- `--from <from>` - The location from which to import. Synth supports multiple import strategies. 
-  
-  Importing from a file: Currently we support importing from JSON files by specifying the path to
-                           the file: `/some/path/to/file.json`.
-  
-  Importing from standard input: Not specifying `from` will accept JSON files from stdin.
 
-  Importing from a database (e.g.
-  postgres): `synth import tpch --from postgres://user:pass@localhost:5432/tpch`
+- `--from <uri>` - The location from which to import. Synth uses Uniform Resource Identifiers (URIs) to define interactions with databases and the filesystem. `<uri>` must therefore be a valid RFC 3986 URI.
+
+  It is possible to import from a file using the URI schemes `json:` and `jsonl:` depending of course on whether the data specified is encoded as JSON or JSON Lines respectively. For example, one could import data from a file `data.jsonl` in the current working directory by specifying `jsonl:data.json`. Note the lack of `//` that you may be used to seeing - this can be omitted as this URI will never have an authority component (unlike, for example, a database URI).
+
+  Data can be imported from standard input by simply not specifying a path in the URI (e.g. `json:` will read JSON data from standard input while `jsonl:` will read JSON Lines data). If no `--from` argument is specified, JSON data will read from standard input by default.
+
+  Importing from a database is possible using the standard URI format of the respective database. For example: `postgres://user:pass@localhost:5432/tpch`
+
+  When dealing with JSON Lines and not specifying a single collection with the `--collection` argument, each generated object is tagged with the name of the collection it was generated from. By default, this is done by adding a property `type` to the object (e.g. `"type": "collection_name"`). The name of this property can be changed using an additional parameter `collection_field_name` added at the end of the URI like so: `jsonl:file.jsonl?collection_field_name=foobar` - with this URI used with `--from`, generate objects will instead have a property like `"foobar": "collection_name"`.
```

Comment:
It was indeed! I haven't used CouchDB before but based on a quick Google search it looks relatively straightforward to write a shell script to import JSON Lines into a database - could likely work well in CI.

Reply:
Is it necessary to support the `collection_field_name=foobar` query? I prefer leaning into more UNIXy solution to this. Why not just support pipe input and utilize something like `jq` for this kind of data manipulation instead? It is also way more powerful than this and avoids Not Invented Here problem

Reply:
@hbina To tell the truth, I would be in favour of relying more heavily of Unix pipes, etc. in general but enough users are on Windows where such operations are awkward that we decided against it. Also specifically with regards to `collection_field_name`, I think it would be useful to set that in advance (rather than modify it afterwards with `jq`) as you may for example have a schema that produces an object with a property `"type"`, in which case you probably don't want Synth overwriting that data, or silently renaming the property's key, or otherwise.